src/app/MessageDef/AttributeDataList.h

```diff
@@ -66,6 +66,13 @@ class Builder : public ListBuilder
      */
     AttributeDataElement::Builder & CreateAttributeDataElementBuilder();
 
+    /**
+     *  @brief Get reference to AttributeDataElement::Builder
```

Comment:
I am unsure if this comment provides any valuable insights into the code.

Could we either rewrite it such that it explains things not obvious from reading code or deleting it? `@return` re-reads the code for me and the `@brief` does not seem to do much either.

Reply:
just delete it, thanks